build/FrameworkPackage/CreateVpackFromNupkg.ps1

```diff
@@ -93,6 +93,9 @@ if($LASTEXITCODE)
     exit
 }
 
+$infoTextFilePath = Join-Path $outputPath "info.txt"
+Out-File -FilePath $infoTextFilePath -InputObject "Vpack created from nuget package: $nupkgFileNameWithoutExtension"
```

Comment:
Should we also leave the source commit ID breadcrumb? Would that be useful?

Reply:
I don't think that would be useful. And we don't even know what the commit is, since we run this script against a nupkg downloaded from nuget.org. So the commit id that the repo from which we are running the script is at is not really useful.